tests/build-workflows.py

```diff
@@ -9,12 +9,18 @@
 from pathlib import Path
 from typing import List
 
+root_path = Path(__file__).parent.resolve()
 
-def subdirs(root_dirs: List[str]) -> List[Path]:
-    dirs: List[Path] = []
-    for r in root_dirs:
-        dirs.extend(Path(r).rglob("**/"))
-    return [d for d in dirs if not (any(c.startswith("_") for c in d.parts) or any(c.startswith(".") for c in d.parts))]
+
+def subdirs() -> List[Path]:
+    result: List[Path] = []
+    for r in testconfig.root_test_dirs:
+        for d in Path(root_path / r).rglob("**/"):
```

Comment:
it seems much safer if we rely on `pytest` to do the globbing, rather than this file having to list all the tests. richard brought this up, that we should just specify the `tests` directory, or perhaps `tests/test_*.py`

Reply:
Yeah that's probably better but rather out of scope of this PR.